components/engine_traits/src/raft_engine.rs

```diff
@@ -138,6 +144,10 @@ pub trait RaftEngine: RaftEngineReadOnly + PerfContextExt + Clone + Sync + Send
     where
         F: FnMut(u64) -> std::result::Result<(), E>,
         E: From<Error>;
+
+    fn store_version(&self) -> Result<Option<StoreVersion>>;
```

Comment:
Should implement for log batch?

Reply:
Why?

Reply:
I mean `put_store_version`.

Reply:
Not necessary, it will be called only once after restart.